src/main/java/com/squareup/kotlinpoet/Util.kt

```diff
@@ -98,15 +98,15 @@ internal fun stringLiteralWithQuotes(value: String): String {
         i += 2
       } else if (c == '\n') {
         // Add a '|' after newlines. This pipe will be removed by trimMargin().
-        result.append("\n|")
+        result.append("\n" + indent + "|")
```

Comment:
String concatenation shouldn't occur in calls to `append` on a `StringBuilder`. Use multiple calls to `append`

Reply:
Will do. Didn't want to go with `\n$indent|` as it's unreadable.